.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -2,21 +2,33 @@
 name: Feature
 about: Suggest an idea for this project
 title: 'MODULE: Neues Feature'
-labels: ''
+labels: feature
 assignees: ''
 
 ---
 
-Neben der Adresse soll neu auch ...
+Als PERSON_DIE_DAMIT_ZU_TUN_HAT,
+möchte ich EIN_FEATURE_HABEN,
+um EIN_ZIEL_ZU ERREICHEN
+
+Neu soll ...
 
 # Mockup
 
 hier folgt ein Mockup ...
 
 # Tech-Spec
 
-- https://github.com/hitobito/hitobito
+- sinngebende Zielrichtung
+- besondere/bekannten Bedingungen und Grenzfälle
+- technische Einschränkungen
+- Ein- und Ausgabeformate
```

Comment:
Allenfalls noch "Schritt-für-Schritt Umsetzungsplan" o.ä. ergänzen? Ich schreibe das jeweils in die Tech Spec wenn ich ein Ticket refine.